src/services/user/services/AdminUsers.js

```diff
@@ -145,7 +146,10 @@ const adminHookGenerator = (kind) => ({
 		get: [hasSchoolPermission(`${kind}_LIST`)],
 		create: [hasSchoolPermission(`${kind}_CREATE`)],
 		update: [hasSchoolPermission(`${kind}_EDIT`)],
-		patch: [hasSchoolPermission(`${kind}_EDIT`)],
+		patch: [
+			hasSchoolPermission(`${kind}_EDIT`),
+			updateAccountUsername,
```

Comment:
I would put this in an afterhook, since the main operation might fail.

Reply:
done, thank you